readme.md

```diff
@@ -1419,6 +1419,14 @@ Categories: `apocalypse` `aliens` `action`
 
 > American nuclear weapons testing results in the creation of a seemingly unstoppable, dinosaur-like beast.
 
+#### [Godzilla vs. Mechagodzilla (1974)](https://www.imdb.com/title/tt0071565/) (1974) [6.4]
+
+Categories: `action` `fantasy` `horror`
+
+> Ape-like aliens build a robotic Godzilla to destroy Japan, and the true Godzilla may not be powerful enough to destroy it. 
+
+Aliens attack Godzilla with a metallic clone of himself. The film features an adventurous spy sub-plot on a cruise from Japan to Okinawa, giant robots, mystical monsters and radioactive beasts -- and the film is not without its charm. - [@bluewhaledelta] (https://github.com/bluewhaledelta)
```

Comment:
Move this above the description?

Reply:
This was not resolved.

Reply:
Also - why do you think this is an awesome film?

Reply:
Ok, yes, line 1428 can be moved to be above the description.

The film (in my opinion) was good as it was the first appearance of Mechagodzilla, and had all the Godzilla hallmarks: giant monsters, explosions, fights, and destruction - also, the dramatic scenes were more fun as they were of the spy/adventure variety.

Reply:
Awesome! Do you need help learning how to edit a PR so you can make these changes, so that this can be merged?

Reply:
Oh! Yes, I would like to learn how to do that.

Reply:
Option 1 should help! https://www.burntfen.com/2015-10-30/how-to-amend-a-commit-on-a-github-pull-request